services/activities.js

```diff
@@ -60,3 +61,23 @@ exports.addActivity = async (data) => {
     throw Error(err.message)
   }
 }
+exports.updateActivity = async (id, name, image, content) => {
+  try {
+    const activity = await Activity.findByPk(id)
+
+    if (!activity) {
+      throw Error(`ID ${id} not found.`)
+    }
+    activity.set({
+      id, name, image, content,
+    })
+    activity.save()
+
+    return activity
+  } catch (error) {
+    if (error.message === 'ID not found.') {
+      throw new ErrorObject('activity not found with that ID', 404)
```

Comment:
Fijate de cambiarle al nuevo handler `throw Error('activity not found with that ID', 404)`